core/src/main/scala/quasar/destination/avalanche/AvalancheConfig.scala

```diff
@@ -44,6 +44,7 @@ final case class AvalancheConfig(
   containerName: ContainerName,
   connectionUri: URI,
   password: ClusterPassword,
+  writeMode: Option[WriteMode],
```

Comment:
What do you think about removing `Option` here and encapsulating the previous behavior in the json codec or maybe a smart constructor? This way we only have a single place where we define `None = Replace`.